src/altinn-app-frontend/webpack.common.js

```diff
@@ -28,11 +28,7 @@ module.exports = {
         },
       },
       {
-        test: /\.scss$/,
-        use: ['style-loader', 'css-loader'],
-      },
-      {
-        test: /\.css$/,
+        test: /\.(sa|sc|c)ss$/,
```

Comment:
This can be changed back, and we can remove the `scss` test entirely. It requires additional loaders (f.ex sass-loader) which we havent set up. And we're not using sass, so no need to keep it here.

```suggestion
        test: /\.css$/,
```